packages/wonder-blocks-data/src/util/memory-cache.js

```diff
@@ -1,134 +0,0 @@
-// @flow
-import type {ValidData, CacheEntry, Cache} from "./types.js";
-
-function deepClone<T: {...}>(source: T | $ReadOnly<T>): $ReadOnly<T> {
-    /**
-     * We want to deep clone the source cache to dodge mutations by external
-     * references. So we serialize the source cache to JSON and parse it
-     * back into a new object.
-     *
-     * NOTE: This doesn't work for get/set property accessors.
-     */
-    const serializedInitCache = JSON.stringify(source);
-    const cloneInitCache = JSON.parse(serializedInitCache);
-    return Object.freeze(cloneInitCache);
-}
```

Comment:
This is replaced by wonder-stuff-core's `clone`. This is a change in the cloning since it does far more than just stringify/parse.